src/components/Callout/MucCallout.vue

```diff
@@ -40,13 +42,33 @@ type calloutType = "info" | "warning" | "success" | "error";
 
 const props = withDefaults(
   defineProps<{
+    /**
+     * Type of the callout, default is `info`.
+     *
+     * Available are `info`, `warning`, `success` and `error`.
+     */
     type?: calloutType;
   }>(),
   {
     type: "info",
   }
 );
 
+defineSlots<{
+  /**
+   * Icon shown above the callout. Defaults to Icons matching the type.
+   */
+  icon(): any;
+  /**
+   * Heading of the callout.
+   */
+  header(): any;
+  /**
+   * Content beneath the Heading shown as text.
```

Comment:
```suggestion
   * Icon shown above the callout. Defaults to icons matching the type.
   */
  icon(): any;
  /**
   * Heading of the callout.
   */
  header(): any;
  /**
   * Content beneath the heading shown as text.
```
Two words in lowercase: icons and heading